### PR TITLE
Added a system to show all the events on a given room, according to user preference

### PR DESCRIPTION
**Pull request by McOmghall**
Please take a look, feedback is always welcome!

### CODE REVIEW

**Comment by ara4n** [comment]
@McOmghall i have no idea how this has got so lost; sorry!

File: src/components/views/rooms/EventTile.js [hunk 1]
```diff
@@ -66,6 +69,8 @@ module.exports = React.createClass({
 
     statics: {
         haveTileForEvent: function(e) {
+            var event = MatrixClientPeg.get().getRoom(e.getRoomId()).getAccountData("org.matrix.room.dev_options");
```
Comment (t3chguy) [inline src/components/views/rooms/EventTile.js]: should probably use the const out of the other file for consistency

**Comment by jryans** [comment]
Thanks for making this contribution a while back. Since the code base has changed since this was opened, it no longer applies cleanly, and I don't think there's a need to keep it open in this state, as we can always find the code here again if needed. If you are still interested in pursuing this feature, please discuss with us in #riot-dev:matrix.org to find a good approach forward.

**Comment by ara4n** [comment]
this will also be rendered unnecessary by MSC1767 (hopefully), which allows arbitrary events to get displayed in a room.

**Pull request closed without merging**